experimental/PropertyDDS/packages/property-properties/src/properties/stringProperty.js

```diff
@@ -34,9 +34,16 @@ var NO_DIRTY_AND_PENDING_SET_TO_PROPERTY_VALUE = {
 };
 
 var DIRTY_AND_NO_PENDING_SET_TO_PROPERTY_VALUE = {
-    pending: 'setAsLiteral',
-    dirty: undefined,
-    flags: MODIFIED_STATE_FLAGS.DIRTY
+    pending: undefined,
+    dirty: 'setAsLiteral',
+    _flags: MODIFIED_STATE_FLAGS.DIRTY,
+    set flags(flags) {
```

Comment:
Should we keep this debug code?

Reply:
let's keep it as far as its in the experimental folder, otherwise it would be hard to debug in case we missed few use-cases in the tests.